data/countries/NZ.yaml

```diff
@@ -10,14 +10,15 @@ holidays:
       - Pacific/Auckland
       - Pacific/Chatham
     days:
-      01-01 and if saturday then next monday if sunday then next tuesday:
+      01-01 and if saturday, sunday then next monday:
         substitute: true
         _name: 01-01
-      01-02 and if saturday,sunday then next monday:
+      01-02 and if saturday then next monday if sunday,monday then next tuesday:
```

Comment:
There is a problem here. If New Years Day is Sunday, The Day After New Years Day should still be on Monday, and the substitute for New Years Day should occur on the Tuesday.

Reply:
I have removed the erroneous substitution of The Day After New Years Day from Monday to Tuesday and updated the tests. Everything looks OK to me now